static/styles/index.scss

```diff
@@ -0,0 +1,94 @@
+:root {
+  font-family: 'Bitter', serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.game-wrapper {
+  background-image: url('../assets/img/QuizBackground.png');
+  background-size: cover;
+  height: 100vh;
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  grid-template-rows: 2fr 4fr 1fr;
+
+  .header {
+    outline: yellow dotted 5px;
+    grid-column: 1/3;
+    grid-row: 1/2;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &__game-logo {
+      width: 30%;
+      height: 50%;
+      background: grey;
+    }
+    &__game-nav {
+      width: 60%;
+      height: 80%;
+      background: rgb(130, 48, 0);
+    }
+  }
+  .game-image {
+    outline: black dotted 5px;
+    grid-column: 1/2;
+    grid-row: 2/3;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &__content {
+      width: 60%;
+      height: 60%;
+      outline: yellow dotted 5px;
+    }
+  }
+
+  .game {
```

Comment:
You could put &-image and &-wrapper also inside in the general .game bracket, the .game-wrapper class doesnt need to wrap all the other classes.